<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportRbService_Wizard_1</class>
 <widget class="QDialog" name="ExportRbService_Wizard_1">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Список записей</string>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout">
     <item>
      <widget class="QCheckBox" name="checkExportAll">
       <property name="text">
        <string>Выгружать всё</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QSplitter" name="splitterTree">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="CTableView" name="tblItems">
      <property name="whatsThis">
       <string comment="ура!">список записей</string>
      </property>
      <property name="tabKeyNavigation">
       <bool>false</bool>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QGroupBox" name="gbFilter">
      <property name="title">
       <string>Фильтр</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="8" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>по</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="edtFilterEndDate">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="7" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>с</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="edtFilterBegDate">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="chkFilterCode">
         <property name="text">
          <string>Код начинается с</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="edtFilterCode">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QCheckBox" name="chkFilterNomenclature">
         <property name="text">
          <string>Номенклатура</string>
         </property>
         <property name="tristate">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QCheckBox" name="chkFilterEIS">
         <property name="text">
          <string>Унаследованно из ЕИС</string>
         </property>
         <property name="tristate">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QCheckBox" name="chkFilterPeriod">
         <property name="text">
          <string>Период</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="10" column="0">
        <widget class="QDialogButtonBox" name="bbxFilter">
         <property name="standardButtons">
          <set>QDialogButtonBox::Apply|QDialogButtonBox::Reset</set>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLineEdit" name="edtFilterName">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="chkFilterName">
         <property name="text">
          <string>Наименование начинается с</string>
         </property>
        </widget>
       </item>
      </layout>
      <zorder>chkFilterCode</zorder>
      <zorder>edtFilterCode</zorder>
      <zorder>chkFilterEIS</zorder>
      <zorder>chkFilterNomenclature</zorder>
      <zorder>chkFilterPeriod</zorder>
      <zorder>bbxFilter</zorder>
      <zorder>verticalSpacer</zorder>
      <zorder>edtFilterName</zorder>
      <zorder>chkFilterName</zorder>
     </widget>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnSelectAll">
       <property name="text">
        <string>Выбрать все</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClearSelection">
       <property name="text">
        <string>Очистить</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QStatusBar" name="statusBar">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>A status bar</string>
     </property>
     <property name="whatsThis">
      <string>A status bar.</string>
     </property>
     <property name="sizeGripEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CTableView</class>
   <extends>QTableView</extends>
   <header>library/TableView.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>checkExportAll</tabstop>
  <tabstop>tblItems</tabstop>
  <tabstop>chkFilterCode</tabstop>
  <tabstop>edtFilterCode</tabstop>
  <tabstop>chkFilterEIS</tabstop>
  <tabstop>chkFilterNomenclature</tabstop>
  <tabstop>chkFilterPeriod</tabstop>
  <tabstop>edtFilterBegDate</tabstop>
  <tabstop>edtFilterEndDate</tabstop>
  <tabstop>bbxFilter</tabstop>
  <tabstop>btnSelectAll</tabstop>
  <tabstop>btnClearSelection</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
